plugins/draw-tools.js

```diff
@@ -77,6 +77,109 @@ window.plugin.drawTools.setDrawColor = function(color) {
   });
 }
 
+// Easy create a draw from an IITC-plugin, allow passing an optional color parameter
+
+// common function used by folloing functions to trigger display on new drawn item
+window.plugin.drawTools.fireDraw = function(layer, layerType) {
```

Comment:
This wrapper is excessive.
I'd prefer to use original code (it is even more expressive).